example/main.dart

```diff
@@ -95,4 +95,40 @@ void main() async {
           postResposne.data);
     });
   });
+
+  group('Raising the custrom Error onRequest', () {
+    const path = 'https://example.com';
+    test('Test that throws raises custom exception', () async {
```

Comment:
Newline above.

Reply:
should I delete it ?"newline"

Reply:
I meant to put a newline in-between line 100 and 101 for readability purposes.

Reply:
ah okay.